### PR TITLE
release 1.5.0

### PR DESCRIPTION
**Pull request by mwrock**
Will merge and release Monday. Made a few changes/additions to the readme so please give a shout if anything looks weird or confusing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,10 +168,10 @@ Once you have the dependencies, you can run the unit tests with `rake`:
 $ bundle exec rake spec
 ```
 
-To run the integration tests you will need a Windows box with the WinRM service properly configured. Its easiest to use a Vagrant Windows box.
+To run the integration tests you will need a Windows box with the WinRM service properly configured. Its easiest to use a Vagrant Windows box (mwrock/Windows2012R2 is public on [atlas](https://atlas.hashicorp.com/mwrock/boxes/Windows2012R2) with an evaluation version of Windows 2012 R2).
 
 1. Create a Windows VM with WinRM configured (see above).
-2. Copy the config-example.yml to config.yml - edit this file with your WinRM connection details.
+2. Edit the config-example.yml with your WinRM connection details.
```
Comment (sneal) [inline README.md]: You can edit the config-example.yml but then it'll show it as dirty when you run `git status` while config.yml is in the .gitignore so it won't.

File: changelog.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # WinRM Gem Changelog
 
+# 1.5.0
+- Deprecating `WinRM::WinRMWebService` methods `cmd`, `run_cmd`, `powershell`, and `run_powershell_script` in favor of the `run_cmd` and `run_powershell_script` methods of the `WinRM::CommandExecutor` class. The `CommandExecutor` allows multiple commands to be run from the same WinRM shell providing a significant performance improvement when issuing multiple calls.
+- Added an `:ssl_peer_fingerprint` option to be used instead of `:no_ssl_peer_verification` and alows a specific certificate to be verified.
```
Comment (sneal) [inline changelog.md]: missing an 'l' in 'allows'

**Comment by sneal** [comment]
@mwrock I like it! Thanks for all your work on this.

File: README.md (hunk 1 above)
Comment (mwrock) [inline README.md]: ha! I missed that. Thanks. I'll add back.

**Pull request merged**